### PR TITLE
A few notifications.

### PR DESCRIPTION
**Pull request by Anonymioo**
Added 4 Notifications.
Not sure about last one.

### CODE REVIEW

File: Monika After Story/game/zz_windowreacts.rpy [hunk 1]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
 
 
 #START: Window Reacts
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_pinterest",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]: As of #4375, adding a consistent label format for these.
```suggestion
            eventlabel="mas_wrs_pinterest",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 2]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            category=['pinterest'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_pinterest
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_pinterest
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 3]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    call display_notif(m_name,"Something new today?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('monika_pinterest')
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_pinterest')
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 4]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_duolingo",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            eventlabel="mas_wrs_duolingo",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 5]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            category=['duolingo'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_duolingo
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_duolingo
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 6]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    call display_notif(m_name,"How is it with your progress in language, [player]?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_duolingo')
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_duolingo')
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 7]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_wikipedia",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            eventlabel="mas_wrs_wikipedia",
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 8]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            category=['wikipedia'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_wikipedia
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_wikipedia
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 9]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    call display_notif(m_name,"Wanna learn something new?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_wikipedia')
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
        $ mas_unlockFailedWRS('mas_wrs_wikipedia')
```

**Comment by Anonymioo** [comment]
Wrong branch.

**Comment by multimokia** [comment]
FYI, you can rebase without closing

**Comment by multimokia** [comment]
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it

File: Monika After Story/game/zz_windowreacts.rpy [hunk 10]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="reddit_rule34",
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]: One thing about the other one was that it was supposed to be *specifically* for Monika related searches, and not general. The expressions for the `show`s here don't really work if you're potentially looking at one of the others.

Comment (Anonymioo) [reply Monika After Story/game/zz_windowreacts.rpy]: > One thing about the other one was that it was supposed to be specifically for Monika related searches, and not general. The expressions for the shows here don't really work if you're potentially looking at one of the others.

I got it. I just tried to make it for Reddit version.

File: Monika After Story/game/zz_windowreacts.rpy [hunk 11]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_pinterest",
+            category=['pinterest'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_pinterest
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_pinterest:
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 12]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_duolingo",
+            category=['duolingo'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_duolingo
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_duolingo:
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 13]
```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_wikipedia",
+            category=['wikipedia'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_wikipedia
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
label mas_wrs_wikipedia:
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 14]
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
 
 
 #START: Window Reacts
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_pinterest",
+            category=['pinterest'],
+            rules={"skip alert": None},
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 15]
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+        $ mas_unlockFailedWRS('mas_wrs_pinterest')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_duolingo",
+            category=['duolingo'],
+            rules={"skip alert": None},
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

File: Monika After Story/game/zz_windowreacts.rpy [hunk 16]
```diff
@@ -320,6 +321,66 @@ label display_notif(title, body, group=None, skip_checks=False):
...
+        $ mas_unlockFailedWRS('mas_wrs_duolingo')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_wikipedia",
+            category=['wikipedia'],
+            rules={"skip alert": None},
```
Comment (multimokia) [inline Monika After Story/game/zz_windowreacts.rpy]:
```suggestion
            rules={"notif-group": "Window Reactions", "skip alert": None},
```

**Comment by multimokia** [comment]
Putting don't merge on this until #4398 gets in

**Comment by multimokia** [comment]
Okay. Will need you to update to the new usage, and possibly add some more variants to the responses since now you can just feed the notif function a list of things you want. (one will be picked at random)

**Comment by Anonymioo** [comment]
> Okay. Will need you to update to the new usage, and possibly add some more variants to the responses since now you can just feed the notif function a list of things you want. (one will be picked at random)

Updated. Added some variants.

**Comment by Anonymioo** [comment]
@multimokia 
I found typo. Or it's normal?
`"Learning new ways to say 'I love you,' [player]?",`

**Comment by multimokia** [comment]
That's normal. Punctuation goes inside quotations.

**Comment by Anonymioo** [comment]
> That's normal. Punctuation goes inside quotations.

Pardon. Didn't knew.

**Comment by multimokia** [comment]
No worries

**Pull request merged**